checkout/react/component/Menu/Desktop/styles.scss

```diff
@@ -137,12 +213,43 @@
     }
   }
 
+  .brand-active {
+    background-color: #fff;
+    font-family: Poppins, sans-serif;
+
+    a {
+      font-weight: 700 !important;
+      font-family: Poppins, sans-serif;
+    }
+  }
+
+  .categories_group__subcate_all-brands {
+    font-weight: 700;
+    text-transform: uppercase;
+    font-size: 1.3rem;
+    padding: 12px 16px;
+    list-style: none;
+    display: flex;
+
+    &::after {
+      content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 477.175 477.175' class='svg-icon'%3E%3Cpath d='M360.731 229.075l-225.1-225.1c-5.3-5.3-13.8-5.3-19.1 0s-5.3 13.8 0 19.1l215.5 215.5-215.5 215.5c-5.3 5.3-5.3 13.8 0 19.1 2.6 2.6 6.1 4 9.5 4 3.4 0 6.9-1.3 9.5-4l225.1-225.1c5.3-5.2 5.3-13.8.1-19z'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
Can we extract and save it in assets folder?

Reply:
Done, an assets/icons folder was created